proof/refine/ARM_HYP/Tcb_R.thy

```diff
@@ -1791,6 +1835,15 @@ where
 | "tcb_inv_wf' (tcbinvocation.SetFlags ref clears sets)
              = (tcb_at' ref and ex_nonz_cap_to' ref)"
 
+lemma invokeSetFlags_helper:
+  "flags = word_to_tcb_flags flags' \<Longrightarrow>
+     corres (=) \<top> (K (flags' \<le> mask numFlags))
+       (return [tcb_flags_to_word (flags - word_to_tcb_flags clears' \<union> word_to_tcb_flags sets')])
+       (return [flags' && ~~ clears' || sets' && mask numFlags])"
+  apply clarsimp
```

Comment:
is the clarsimp here necessary?

Reply:
I think it might have been necessary at one point because the word proof was being annoyingly fiddly but it isn't anymore, so I've folded it into one fastforce.